contracts/SystemSettings.sol

```diff
@@ -129,6 +129,30 @@ contract SystemSettings is Owned, MixinSystemSettings, ISystemSettings {
         return getCrossDomainMessageGasLimit(gasLimitType);
     }
 
+    // SIP-115 Atomic exchanges
+    // max allowed volume per block for atomic exchanges
+    function atomicMaxVolumePerBlock() external view returns (uint) {
+        return getAtomicMaxVolumePerBlock();
+    }
+
+    // SIP-115 Atomic exchanges
+    // price dampener for chainlink prices when considered for atomic exchanges
+    function atomicPriceBuffer() external view returns (uint) {
+        return getAtomicPriceBuffer();
+    }
+
+    // SIP-115 Atomic exchanges
+    // time window (in 30min periods) for TWAP prices when considered for atomic exchanges
+    function atomicTwapPriceWindow() external view returns (uint) {
+        return getAtomicTwapPriceWindow();
+    }
+
+    // SIP-115 Atomic exchanges
+    // equivalent asset to use for a synth when considering external prices for atomic exchanges
+    function atomicEquivalentForSynth() external view returns (address) {
```

Comment:
In the current SIP I've documented `synthCanTradeAtomically()` for enabling/disabling whether a synth can be exchanged atomically.

However, with `ExchangeRates.effectiveAtomicValueAndRates()`, we'll want a mapping to go from synth to look up token (e.g. sETH -> WETH, sBTC -> WBTC, etc) for the Keep3r oracle.

So I thought we could do both with the mapping—synths that are allowed to be atomically exchanged will have an equivalent asset mapped through here.